cli.js

```diff
@@ -17,6 +17,7 @@ if (debug.enabled) {
 	require('time-require');
 }
 
+var logUpdate = require('log-update');
```

Comment:
Can you add `logUpdate.stderr();` to hide the cursor as early as possible. Currently it doesn't hide until the tests actually starts running. Should include a comment on why it's there.